awx_collection/plugins/modules/tower_job_wait.py

```diff
@@ -92,6 +104,7 @@
 
 from ..module_utils.ansible_tower import TowerModule, tower_auth_config, tower_check_mode
 from ansible.module_utils.six import PY2
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
Looks like an unused import, the needed logic is obtained from `TowerModule` which has this as a parent class.

Reply:
Ok I will remove that line. I built this feature a few years ago and updated it in January or so. I see that a lot changed since January. I am working on trying to make this work with the new code. 
I see that `awx_collection/plugins/modules/tower_job_launch.py` has some of what I need it it so I won't have to recode too much.
Thank you for this help. I appreciate it.

I am looking at the `module.get_one()` logic right now.

Reply:
@AlanCoding 
I rewrote this to use the new code base. It now builds sucessfully, passing the build checks.